code/game/objects/items/weapons/grenades/chronogrenade.dm

```diff
@@ -10,4 +10,49 @@
 
 /obj/item/weapon/grenade/chronogrenade/prime()
 	timestop(src, duration, radius)
-	qdel(src)
\ No newline at end of file
+	qdel(src)
+
+/obj/item/weapon/grenade/chronogrenade/future
+	desc = "This experimental weapon will send all entities in the local area ten seconds into the future."
+	icon_state = "future_grenade"
+
+/obj/item/weapon/grenade/chronogrenade/future/prime()
+	future_rift(src, duration, radius)
+	qdel(src)
+
+/proc/future_rift(atom/A, var/duration, var/range)	//Sends all non-timeless atoms in range duration time into the future.
+	if(!A || !duration)
+		return
+	if(!range)
+		range = 7
+
+	var/turf/ourturf = get_turf(A)
+	var/list/targets = circlerangeturfs(A, range)
+	spawn()
+		for(var/client/C in clients)
+			if(C.mob)
+				C.mob.see_fall(ourturf, range)
+		spawn(10)
+		for(var/client/C in clients)
```

Comment:
I'm surprised that compiles
Most things that expect to be followed by a tabbed block don't